maven-release-plugin/src/it/projects/prepare/semver-policy/pom.xml

```diff
@@ -21,7 +21,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.maven.plugin.release.it</groupId>
-  <artifactId>oddversion-policy</artifactId>
+  <artifactId>semver-policy</artifactId>
```

Comment:
Is that a bugfix?

Reply:
I think this is a copy-paste error from long ago. It is just naming, doesn't really fix anything.